src/frontend/core/executor/impl/TriangleCountExecutor.cpp

```diff
@@ -319,12 +706,9 @@ long TriangleCountExecutor::getTriangleCount(int graphId, std::string host, int
         if (result_wr < 0) {
             triangleCount_logger.log("Error writing to socket", "error");
         }
-
         triangleCount_logger.log("Sent : " + masterIP, "info");
-        bzero(data, 301);
-        read(sockfd, data, 300);
-        response = (data);
 
+        response = Utils::read_str_trim_wrapper(sockfd, data, 300);
```

Comment:
Fix the issue of 300 here. There are few statements which needs to be fixed.

Reply:
Replaced all 300's with constant

Reply:
Ok